README.md

```diff
@@ -114,3 +114,9 @@ docker build -t litdev . --build-arg PLAYGROUND_SANDBOX=http://localhost:8081/
 docker run --rm --name litdev -p 8080:8080 -e PORT=8080 -e MODE=main litdev
 docker run --rm --name litdev-playground -p 8081:8081 -e PORT=8081 -e MODE=playground litdev
 ```
+
+### Updating snapshot tests
+
+Snapshots that are committed to the repository need to be created through the
+"Artifacts / Download link for updated snapshots" Github Action. This action
+generates a zip archive `golden-results` which can be extracted into `tests/`.
```

Comment:
Maybe provide a sample command? `unzip golden-results.zip -d tests/` or similar?

Reply:
Great suggestion. Manually tested and added command to readme.